phpBB/phpbb/di/container_builder.php

```diff
@@ -364,6 +364,12 @@ protected function load_extensions()
 			$ext_container->register('cache.driver', '\\phpbb\\cache\\driver\\dummy');
 			$ext_container->compile();
 
+			$config = $ext_container->get('config');
```

Comment:
Seems like this is not available in tests which is also why tests currently fail.